docs/snippets/svelte/msw-addon-configure-handlers-graphql.js.mdx

```diff
@@ -37,35 +40,39 @@ const TestData = {
 };
 
 export const MockedSuccess = {
-  render: () => ({
-    Component: DocumentScreen,
-  }),
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(ctx.data(TestData));
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', () => {
+          return new HttpResponse.json({
+            data: {
+              allFilms: {
+                films,
+              },
+            }
+          });
+        }),
+      ],
+    },
   },
 };
 
 export const MockedError = {
-  render: () => ({
-    Component: DocumentScreen,
-  }),
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(
-          ctx.delay(800),
-          ctx.errors([
-            {
-              message: 'Access denied',
-            },
-          ]),
-        );
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', async () => {
+          delay(800);
```

Comment:
This should be awaited
Also don't forget to add `HttpResponse` and `delay` to the msw imports at line 4